app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] Great job team for adding devise to your app. However, some things are working as supposed.
First, while setting up your app on my local machine with this command `rails db:setup` which would create the db on my pc. It fires up confirm your email page on the browser which is not normal. That page is supposed to display after clicking on the `sign up` button on the sign up page to confirm the user's email. Kindly look into your setup properly and fix where the issue is.

### 
![your-setup](https://github.com/shihabrafio/Blog_App/assets/26597432/e28639ec-13d9-44ae-a6be-55e5a1c4c26d)

###
![your-confirm-page](https://github.com/shihabrafio/Blog_App/assets/26597432/e73dddc7-02f8-408c-b7f5-b55df9e1465a)